specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -283,40 +347,127 @@ not required to upload.
 
   @doc("Optional load test config")
   optionalLoadTestConfig?: OptionalLoadTestConfig;
+
+  @added(APIVersions.v2023_04_01_preview)
+  @doc("Type of test. e.g. QuickTest, UrlBasedTest")
+  testType?: TestType;
+}
+
+@doc("Quick test config")
+model QuickTestConfig {
+  @doc("Test URL. Provide the complete HTTP URL. For example, http://contoso-app.azurewebsites.net/login")
```

Comment:
QuckTestConfig did we have in GA or did it come during the public preview?

if public preview do we need to add the tag
` @added(APIVersions.v2023_04_01_preview)` in the model itself

Reply:
This change is under discussion currently with API review team, once this is finalized will add appropriately.